quaddtype/numpy_quaddtype/src/dragon4.c

```diff
@@ -22,6 +25,28 @@ Modifications are specific to support the SLEEF_QUAD
 #include "dtype.h"
 #include "scalar.h"
 
+#if !defined(HAVE_THREAD_LOCAL) && !defined(HAVE__THREAD_LOCAL) && \
+    !defined(HAVE___THREAD) && !defined(HAVE___DECLSPEC_THREAD_) && \
+    !defined(__cplusplus)
+#warning "No thread-local storage support detected! NPY_TLS will be empty, causing thread safety issues."
+#else
+#warning "NPY_TLS Thread-local storage support detected."
```

Comment:
probably delete this though - you can get this information from the meson output and it's not correct to warn if something is working correctly

Reply:
yeah MSVC seems not to support